aws-logs-resourcepolicy/aws-logs-resourcepolicy.json

```diff
@@ -0,0 +1,63 @@
+{
+  "typeName": "AWS::Logs::ResourcePolicy",
+  "description": "The resource schema for AWSLogs ResourcePolicy",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs.git",
+  "properties": {
+    "PolicyName": {
+      "description": "A name for resource policy",
+      "type": "string",
+      "pattern": "^([^:*\\/]+\\/?)*[^:*\\/]+$",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "PolicyDocument": {
+      "description": "The policy document",
+      "type": "string",
+      "pattern": "[\\u0009\\u000A\\u000D\\u0020-\\u00FF]+",
+      "minLength": 1,
+      "maxLength": 131072
```

Comment:
this should be 5120. see max length in [docs](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutResourcePolicy.html#API_PutResourcePolicy_RequestSyntax)